templates/themes/horizon/invenio_communities/details/home/index.html

```diff
@@ -119,19 +119,18 @@ <h1 class="ui large header">{{ _("Recent uploads") }}</h1>
       </div>
     {% endif %}
 
-    <div class="ui stackable theme-font grid container rel-mt-2">
+    <div class="ui stackable theme-font one column grid container rel-mt-2">
```

Comment:
I feel like this one column grid (with only one content column below) is a bit silly and could be removed, but I think we follow a similar pattern of having a containing grid in many other places.